### DISCUSSION TITLE
请问可以提供一个qq群或者微信群吗？

### DISCUSSION THREAD

**Discussion by cxxcunow** [milvus-io/milvus#4156] [Ideas & Feature requests]
请问可以提供一个qq群或者微信群吗？

**Comment by shiyu22** [comment]
You can Follow Us on [the website](https://milvus.io/).
![image](https://user-images.githubusercontent.com/53459423/97945527-c1f74200-1dc2-11eb-83d5-78172cf1c188.png)

**Reply by twtiyb** [reply]
微信群 打不开

**Reply by shiyu22** [reply]
We can do it, please check your WeChat setting.

**Comment by Hannah-cc** [comment]
可以啊

**Reply by sloth2012** [reply]
群满了

**Reply by shiyu22** [reply]
你可以扫第二个二维码，进入公众号后，点下方的工作栏，选择社区动态->技术交流群，然后扫描那个进群二维码就好了。
